README.rst

```diff
@@ -175,7 +177,7 @@ Choose from the list of available fixers:
 * **psr0** [PSR-0]
                 Classes must be in a path that matches
                 their namespace, be at least one
-                namespace deep, and the class name
```

Comment:
But we follow Symfony Coding Standards. So maybe we want to follow the documentation standards, too..
And I can see "Symfony" in the namespace. Or is this for bc reasons only?